sessions/css-animations/demo-end/css/styles.css

```diff
@@ -17,98 +17,96 @@ body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
     Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
-
-  background-color: var(--foam);
-  padding: 20px;
+	background-color: var(--foam);
+	padding: 20px;
 }
 
 main {
-  display: grid;
-  gap: 20px;
+	display: grid;
+	gap: 20px;
 }
 
 h1 {
-  font-size: 3rem;
-  color: var(--nemo);
+	font-size: 3rem;
+	color: var(--nemo);
 }
 
 h2 {
-  margin: 0;
-  color: var(--granite);
+	margin: 0;
+	color: var(--granite);
 }
 
 button {
-  border: none;
-  padding: 14px 20px;
-  cursor: pointer;
-  font-size: 1rem;
-  border-radius: 7px;
-  box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px,
-    rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+	border: none;
+	padding: 14px 20px;
+	cursor: pointer;
+	font-size: 1rem;
+	border-radius: 7px;
+	box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
 }
 
 section {
-  position: relative;
-  padding: 20px;
-  border-radius: 14px;
-  box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px,
-    rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
-  display: flex;
-  flex-direction: column;
-  gap: 1.5rem;
-  align-items: center;
+	position: relative;
+	padding: 20px;
+	border-radius: 14px;
+	box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+	display: flex;
+	flex-direction: column;
+	gap: 1.5rem;
+	align-items: center;
 }
 
 .button--color {
-  color: var(--water);
-  background-color: var(--granite);
-  transition: background-color 0.4s ease-in-out;
+	color: var(--water);
+	background-color: var(--granite);
+	transition: background-color 0.4s ease-in-out;
 }
 
 .button--color:hover {
-  background-color: var(--nemo);
+	background-color: var(--nemo);
 }
 
-.button--shadow {
-  background-color: var(--nemo);
-  color: var(--water);
-  transition: 0.3s ease;
-
-  box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+.button--scale {
+	background-color: #ccccd4;
+	box-shadow: rgba(0, 0, 0, 0.3) 0px 19px 38px;
+	transition: 0.05s linear;
 }
 
-.button--shadow:hover {
-  box-shadow: rgba(0, 0, 0, 0.3) 0px 19px 38px,
-    rgba(0, 0, 0, 0.22) 0px 15px 12px;
+.button--scale:active {
+	transform: scale(0.9);
+	box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
```

Comment:
There is an issue (neuefische/web-curriculum-new-format#140) to refactor all CSS styles; it includes removing rgba color styling and replace it with modern rules.
What do you think, could you update it here as well?

Reply:
we'll fix this for all challenges later